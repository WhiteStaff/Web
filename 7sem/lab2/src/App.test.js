// import React from 'react';
// import {CityCard} from './components/CityCard';
// import renderer from 'react-test-renderer';
// import {SavedCity} from "./components/SavedCity";
// import {shallow} from "../enzyme";
// import {shallowToJson} from "enzyme-to-json";
//
// const json =
//     {
//         data:
//             {
//                 "coord": {"lon": 139, "lat": 35},
//                 "weather": [
//                     {
//                         "id": 800,
//                         "main": "Clear",
//                         "description": "clear sky",
//                         "icon": "01n"
//                     }
//                 ],
//                 "base": "stations",
//                 "main": {
//                     "temp": 289.92,
//                     "pressure": 1009,
//                     "humidity": 92,
//                     "temp_min": 288.71,
//                     "temp_max": 290.93
//                 },
//                 "wind": {
//                     "speed": 0.47,
//                     "deg": 107.538
//                 },
//                 "clouds": {
//                     "all": 2
//                 },
//                 "dt": 1560350192,
//                 "sys": {
//                     "type": 3,
//                     "id": 2019346,
//                     "message": 0.0065,
//                     "country": "JP",
//                     "sunrise": 1560281377,
//                     "sunset": 1560333478
//                 },
//                 "timezone": 32400,
//                 "id": 1851632,
//                 "name": "Shuzenji",
//                 "cod": 200
//             }
//     }
//
// it('CityCard renders correctly', () => {
//
//     const tree = renderer.create(<CityCard json={json.data}/>).toJSON();
//
//     expect(tree).toMatchSnapshot();
// });
//
// it('SavedCity renders correctly', () => {
//     const saved = shallow(<SavedCity key = {1}json={json}/>);
//         saved.SetState({error: true});
//         expect(shallowToJson(saved)).toMatchSnapshot();
//
// });
// /*it('SavedCity renders correctly', () => {
//
//     const tree = renderer.create(<SavedCity key={1} json={json}/>).toJSON();
//
//     expect(tree).toMatchSnapshot();
// });


/*it('renders correctly', () => {

    const tree = renderer.create(<Wrapper />).toJSON();

    expect(tree).toMatchSnapshot();
});*/
