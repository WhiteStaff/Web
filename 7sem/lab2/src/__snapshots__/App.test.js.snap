// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="alert alert-dark row px-0 mx-1"
  >
    <div
      class="col-6"
    >
      Ветер
    </div>
    <div
      class="col-6 text-right"
    >
      0
    </div>
  </div>
  <div
    class="alert alert-dark row px-0 mx-1"
  >
    <div
      class="col-6"
    >
      Облачность
    </div>
    <div
      class="col-6 text-right"
    >
      0
    </div>
  </div>
  <div
    class="alert alert-dark row px-0 mx-1"
  >
    <div
      class="col-6"
    >
      Давление
    </div>
    <div
      class="col-6 text-right"
    >
      0
    </div>
  </div>
  <div
    class="alert alert-dark row px-0 mx-1"
  >
    <div
      class="col-6"
    >
      Влажность
    </div>
    <div
      class="col-6 text-right"
    >
      0
    </div>
  </div>
  <div
    class="alert alert-dark row px-0 mx-1"
  >
    <div
      class="col-6"
    >
      Координаты
    </div>
    <div
      class="col-6 text-right"
    >
      [
      0
      ]
    </div>
  </div>
</div>
`;
